backend/app/tests/seed_data/seed_data.py

```
@@ -0,0 +1,459 @@
+class DocumentData(BaseModel):
+    fname: str
+    object_store_url: str
+    organization_name: str
+    project_name: str
+
+
+def load_seed_data() -> dict:
+    """Load seed data from JSON file."""
+    json_path = Path(__file__).parent / "seed_data.json"
+    try:
+        with open(json_path, "r") as f:
```

Ruff (0.14.8)

Unnecessary mode argument

Remove mode argument

(UP015)